client/src/index.jsx

```diff
@@ -35,83 +37,88 @@ class Menu extends React.Component {
     this.setState({
       selectedMenuIndex: menuIndex
     });
-
-    for (var i = 0; i < document.getElementsByClassName('menu-buttons').length; i++) {
+    for (var i = 0; i < document.getElementsByClassName(styles.menuButtons).length; i++) {
       if (menuIndex === i) {
-        document.getElementsByClassName('menu-buttons')[i].id = 'renderedMenu';
+        document.getElementsByClassName(styles.menuButtons)[i].id = styles.renderedMenu;
       } else {
-        document.getElementsByClassName('menu-buttons')[i].id = '';
+        document.getElementsByClassName(styles.menuButtons)[i].id = '';
       }
     }
-
```

Comment:
an idea to simplify the code:  since you're making multiple style changes to only a handful of DOM elements, maybe assign the DOM elements to variables and modify the variables instead.  that way you don't have to write document.getElementById(...) over and over again.  it may also make sense to put the adjustments inside a helper function.